### PR TITLE
Update all the things

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

**Review by dnkoutso — APPROVED** [review]

File: .rubocop.yml [hunk 1]
```diff
@@ -1,3 +1,108 @@
...
+Style/RedundantArgument: # new in 1.4
+  Enabled: true
+Style/RedundantSelfAssignmentBranch: # new in 1.19
+  Enabled: true
+Style/SelectByRegexp: # new in 1.22
+  Enabled: true
+Style/StringChars: # new in 1.12
+  Enabled: true
+Style/SwapValues: # new in 1.1
+  Enabled: true
```
Comment (orta) [inline .rubocop.yml:108]: I hope that was automated :P

File: app/controllers/html_controller.rb [hunk 2]
```diff
@@ -21,11 +27,9 @@ class HTMLController < AppController
...
       #
       not_found do
-        begin
-          slim :'../html/not_found', :status => 404, :layout => :'../html/layout'
-        rescue Errno::ENOENT
-          slim :'html/not_found', :status => 404, :layout => :'html/layout'
-        end
+        slim :'../html/not_found', :status => 404, :layout => :'../html/layout'
+      rescue Errno::ENOENT
+        slim :'html/not_found', :status => 404, :layout => :'html/layout'
```
Comment (orta) [inline app/controllers/html_controller.rb:32]: kudos, I imagine this can't have been easy to figure out

**Review by orta — APPROVED** [review]
Looks good to me!

File: .rubocop.yml (hunk 1 above)
Comment (segiddins) [reply .rubocop.yml:108]: yeah, copy-paste

**Pull request merged**